app/furniture/marketplace/tax_rates/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<%- breadcrumb(:marketplace_tax_rates, marketplace) %>
+
+<section class="max-w-2xl self-stretch mx-auto">
+  <main>
+    <div class="mt-3 grid grid-cols-1 gap-5 sm:gap-6 sm:grid-cols-2 lg:grid-cols-4">
+      <%= render Marketplace::TaxRateComponent.with_collection(tax_rates) %>
+    </div>
+  </main>
+  <footer class="text-center">
+    <%= link_to(t("marketplace.tax_rates.new.link_to"), marketplace.location(:new, child: :tax_rate)) %>
+  </footer>
+</section>
```

Comment:
Can a screenshot be added to the PR description of what the `TaxRate#index screen` looks like?

Reply:
Yah!